index.html

```diff
@@ -5425,39 +5425,23 @@ <h2 id="AudioWorkletNodeOptions">
               <a><code>AudioWorkletProcessor</code></a> are used to initialize
               the parameter values upon the creation of a node.
             </p>
-            <dl title="dictionary AudioWorkletNodeOptions" class="idl">
+            <dl title="dictionary AudioWorkletNodeOptions : AudioNodeOptions"
+            class="idl">
               <dt>
                 unsigned long numberOfInputs
               </dt>
               <dd>
-                Same to <a>numberOfInputs</a> in <a><code>AudioNode</code></a>.
+                Same to <a href=
```

Comment:
Although you didn't actually change the content of this text, I think it reads better if you said something like

This is used to initialize the value of AudioNode numberOfInputs attribute.

Same comment for numberOfOutputs.